user_tools/src/spark_rapids_tools/enums.py

```diff
@@ -210,3 +210,15 @@ def create_default_model_args(cls, model_type: str) -> dict:
             'xgboostEnabled': model_type == QualEstimationModel.XGBOOST,
             'customModelFile': None,
         }
+
+
+class ExecutionEngine(EnumeratedType):
+    """
+    Represents the execution engine for the application (Spark or Photon).
+    """
+    SPARK = 'spark'
+    PHOTON = 'photon'
+
+    @classmethod
+    def get_default(cls) -> 'ExecutionEngine':
+        return cls.SPARK
```

Comment:
should we have a class defined per platform? All the platforms will be bound to the default enumType. Then DB extends that definition adding the photon type.

Reply:
This enum is no longer required as the logic to parse the runtime has been moved to Scala code.